Sources_v3/StreamChatUI/Common Views/AvatarView.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class AvatarView: UIView {
+    
+    // MARK: - Subviews
+    
+    public let imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
already set in `embed` :D

Reply:
You are right 👍 

But I'd keep this configuration anyway. The reason is simple: if we decide to change the `setupLayout` or override this func in a inheritor it can happen the `embed` is not used -> the layout will be broken